<launch>
    <arg name="num_uavs" default="1"/>
    <arg name="uav_id" default="1"/>

    <group ns="uav$(arg uav_id)">
        <node pkg="frontier_exploration" type="frontier_exploration" name="frontier_exploration" output="screen">
            <param name="num_uavs" value="$(arg num_uavs)"/>
            <param name="uav_id" value="$(arg uav_id)"/>
            <param name="bandwidth" value="1.0"/>
            <param name="comms_range" value="1000.0"/>

            <param name="exploration_max_x" value="56.0" />
            <param name="exploration_max_y" value="56.0" />
            <param name="exploration_max_z" value="3.0" />

            <param name="exploration_min_x" value="32.0" />
            <param name="exploration_min_y" value="32.0" />
            <param name="exploration_min_z" value="0.0" />

            <param name="lambda" value="0.10"/>

            <param name="update_rate" value="0.5"/>
            <param name="max_sensor_range" value="20.0"/>
            <param name="min_sensor_range" value="0.7"/>
            <param name="d_exp" value="14"/>
            <param name="waypoint_distance" value="20.0"/>
        </node>
    </group>
</launch>
